packages/extension-ui/src/util/getFaviconUrl.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+export function getFaviconUrl(url: string): string {
+  try {
+    const domain = new URL(url).hostname;
+
+    return `https://www.google.com/s2/favicons?domain=${domain}`;
+  } catch (error) {
+    console.error('An error occurred while parsing URL:', error);
+
+    return '';
```

Comment:
Do we maybe want to return some default in case no favicon found? Or just empty field?